sdks/go/pkg/beam/core/util/stringx/bytes.go

```diff
@@ -15,9 +15,6 @@
 
 // Package stringx contains utilities for working with strings. It
 // complements the standard "strings" package.
-//
-// Deprecated: the utilities in this package are unused within the code base
-// and will be removed in a future Beam release.
 package stringx
```

Comment:
Did you mean to modify this? Looks like probably a bad merge with Jack's pr - https://github.com/apache/beam/pull/16884/files

Reply:
good catch - reverted